parsers/ESKOM.py

```diff
@@ -53,8 +53,8 @@
 
 def get_url() -> str:
     """Returns the formatted URL"""
-    date = datetime.utcnow()
-    return f"https://www.eskom.co.za/dataportal/wp-content/uploads/{date.strftime('%Y')}/{date.strftime('%m')}/Station_Build_Up.csv"
+    date = datetime.now(timezone.utc)
+    return f"https://www.eskom.co.za/dataportal/wp-content/uploads/{date.strftime('%Y/%m')}/{date.strftime('%m')}/Station_Build_Up.csv"
```

Comment:
Why the URL change here? :)

Reply:
Oh that was just me trying things out, I thought I reverted that...

This does work though but I'll change it back to keep the PR as clean as possible.